examples/features/reflection/server/main.go

```diff
@@ -16,7 +16,8 @@
  *
  */
 
-// Binary server is an example server.
+// Binary server demonstrates how to register multiple services and enable
```

Comment:
there is no client change for it?

Reply:
There is no client file for it

Reply:
The README asks to use `grpcurl` as the client, which makes sense.